### PR TITLE
Fix plots and improve code organization

### PR DESCRIPTION
**Pull request by ufechner7**
- [x] Add unit tests on Mac OS
- [x] Add page "Examples" to the documentation
- [x] Fix outdated docstrings
- [x] Add select_measurement() to menu
- [x] Add function plot_x for remote plotting
- [x] Add field fps to vis_default.yaml and the Vis struct and the other vis yaml files and make use of it
- [x] fix plot_measurements for 54 turbine case by using a nxm plot (n lines in m, stacked plots)
- [x] Add tests for the 54 turbine plot
- [x] Improve menu.jl
- [x] Refactoring of runFLORIS! to break it up into smaller functions
- [x] Add comments at the top of the files in the floridyn and floris folders to explain the file content of the files
- [x] Also add such a comment to settings.jl
- [x] Rename operation point to observation point in different variations
- [x] Add test for empty files

### CODE REVIEW

File: src/visualisation/plot_measurements.jl [hunk 1]
```diff
@@ -68,64 +78,65 @@ function plotMeasurements(plt, wf::WindFarm, md::DataFrame, vis::Vis; separated=
...
+                plt.tight_layout()
+                fig.subplots_adjust(wspace=0.55)
+            end
+        else
+            try
+                times, plot_data, turbine_labels, subplot_labels = prepare_large_plot_inputs(wf, md, data_column, ylabel)
+                plot_x(times, plot_data...; ylabels=turbine_labels, labels=subplot_labels,
+                       fig=title, xlabel="rel_time [s]", ysize=9, bottom=0.02, pltctrl, legend_size=6, loc="center left")
+            catch e
+                println("Error in plot_x: $e")
```
[the remark this answers is not in the document]
Comment (ufechner7) [reply src/visualisation/plot_measurements.jl]: Applied suggested change.
